components/Avatar/style.css

```diff
@@ -0,0 +1,50 @@
+.avatar{
+    border-radius:50%;
+  }
+  .square-avatar{
+    border-radius:0.5rem;
+  }
+  .avatar-sm{
+    width:3rem;
+  }
+  .avatar-md{
+    width:5rem;
+  }
+  .avatar-lg{
+    width:8rem;
+  }
+  .letter-avatars{
+  
+    background-color:green;
+    padding:0.2rem;
+  }
+  .material-icons.icon-avatar{
+    color:black;
+  }
+  .material-icons .Icon-avatar{
+    background-color:green;
+  }
```

Comment:
The two `icon-avatar` and `Icon-avatar` classes have the same name just they are different cased. You could change the name of anyone as it's creating a bit of confusion here. I would suggest changing the `Icon-avatar` in order to maintain consistency of small cased naming.

Reply:
okay i resolve that